README.md

```diff
@@ -39,17 +42,18 @@ your platform.
 
 Verify that you can start `ankerctl.py`, and get the help screen:
 
+#### For Windows, use: 
 ```powershell
-## For Windows, use:
 python3 ankerctl.py -h
 ```
+If that does not work then try running using ```python ankerctl.py -h```
```

Comment:
Triple-backticks are for block comments. For in-line code quotes, use single backticks.

Reply:
Fixed